src/users/users.interface.ts

```diff
@@ -4,10 +4,16 @@ export enum Role {
   SUPERVISOR = 'SUPERVISOR'
 }
 
+export enum Status {
+  NEW = 'NEW',
```

Comment:
Better to change to a name that will point that the user is unregistered.

Reply:
I think NEW is fine. We can use more informative version like REGISTRATION_STARTED